docs/units/equivalencies.rst

```diff
@@ -502,6 +502,22 @@ In a special relativity context it can be convenient to use the
 
 .. EXAMPLE END
 
+Mass-Length-Time Equivalency
```

Comment:
Given that there are many types of geometrized unit systems, I think this should be renamed.